website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -343,6 +345,25 @@ function MyPlayground(props) {
 }
 ```
 
+### Configuration
+
+You can also configurate `@docusaurus/theme-live-codeblock` plugin with the following parameters:
```

Comment:
going to move this part to the API doc of the plugin because it's a less useful option